etna/search/tests/test_forms.py

```diff
@@ -94,5 +94,5 @@ def test_opening_start_date_before_opening_end_date_is_invalid(self):
                 self.assertFalse(form.is_valid(), label)
                 self.assertEqual(
                     form.errors.get("opening_start_date", None),
-                    ["There is a problem. Start date cannot be after end date"],
+                    ["There is a problem. Start date cannot be after end date."],
```

Comment:
There is test [error](https://github.com/nationalarchives/ds-wagtail/actions/runs/3684403561/jobs/6234101523#step:9:57)  `AssertionError: ['There is a problem. Start date cannot be after end date'] != ['There is a problem. Start date cannot be after end date.']` is failing because there is a difference of full stop at the end of the sentence.
I recommend 
1. update test case to use Error Message from Constant [here](https://github.com/nationalarchives/ds-wagtail/blob/b874aa84b2ea95cbd3c6942413a84b71cf20eb18/etna/ciim/constants.py#L614) 
2. Update the Constant to end with a full-stop. This will then reflect both on _front end UI_ as well as _unit tests_

Would you like to do this or I can pick this up?

Reply:
Thanks @TNA-Allan if you could apply the fix, it would be appreciated. The version with the full stop is the correct one

Reply:
Done, @JohnHeeryTNA , please can you pull the latest changes